lerna.json

```diff
@@ -16,7 +16,10 @@
   "npmClient": "yarn",
   "command": {
     "version": {
-      "allowBranch": "master"
+      "allowBranch": [
```

Comment:
remove. I'll have a task for this soon

Reply:
can you link the ticket and task here please?

Reply:
Doesn't exist yet.